ENVI_ImageBandDifference.md

```diff
@@ -50,43 +52,43 @@ Mandatory (optional) settings are listed as Required = True (Required = False).
-input_raster2_band_grouping|False|None| [More on ENVI input port](insert links here)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
-input_raster2_filename|False|None| |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
+input_raster1|True|None| A valid S3 folder containing image files. |Specify a single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
+input_raster1_format|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the format of the image, for example: landsat-8. -- Value Type: STRING
+input_raster1_band_grouping|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
+input_raster1_filename|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster) |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
+input_raster2|True|None|  A valid S3 folder containing image files.  |Specify a second single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
+input_raster2_format|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the format of the image, for example: landsat-8. -- Value Type: DICTIONARY
+input_raster2_band_grouping|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
+input_raster2_filename|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster) |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
 output_raster_uri_filename|False|None| |Specify a string with the fully-qualified path and filename for OUTPUT_RASTER. -- Value Type: STRING
 
 ### Outputs
 The following table lists all taskname outputs.
 Mandatory (optional) settings are listed as Required = True (Required = False).
 
-  Name  |  Required  |  Default  |  Valid Values  |  Description  
---------|:----------:|-----------|----------------|---------------
-task_meta_data|False|None| |GBDX Option. Output location for task meta data such as execution log and output JSON
-output_raster_uri|True|None| |Output for OUTPUT_RASTER. -- Value Type: ENVIURI
+  Name  |  Required  |  Default  |  Description  
```

Comment:
@creeder23 I removed the `Valid Values` column because it is not applicable to outputs. However, it changes the consistency for all the other task docs. So feel free to delete this change.